esx_service/vmdk_ops_test.py

```diff
@@ -731,7 +740,7 @@ def setUp(self):
                 self.datastore_path = datastore[2]
                 logging.debug("datastore_name=%s datastore_path=%s", self.datastore_name,
                                                                      self.datastore_path)
-                if len(datastores) >= 1:
+                if len(datastores) >1:
```

Comment:
So if we have only one datastore, `datastore1`, `self.datastore1_name ` and  `self.datastoer1_path ` will be not set. Is it OK ? '
'also - nit: space after `>` is missing